biggraphite/accessor.py

```diff
@@ -91,7 +91,7 @@ def on_done(exception):
         event.set()
 
     async_function(*args, on_done=on_done, **kwargs)
-    event.wait()
+    event.wait(3.0)
```

Comment:
can you add a code comment explaining why, with the link to the issue ?
Thanks !